### ISSUE TITLE
class-no-dup false positive

### ISSUE THREAD

**Issue by buu700** [htmllint/htmllint#180]
class-no-dup incorrectly reports a failure when a class attribute has whitespace on both ends.

Examples:
```
<div class='
	a b c d e f g h i j k l m
	n o p q r s t u v w x y z
'></div>

<div class=' a b c d e f g h i j k l m n o p q r s t u v w x y z '></div>
```

However, `<div class='a b c d e f g h i j k l m n o p q r s t u v w x y z'></div>` works as expected.

**Comment by mlochbaum** [comment]
Thanks for the report!

**Comment by buu700** [comment]
No problem, thanks for the quick fix!
